service/worker/batcher/workflow.go

```diff
@@ -59,6 +61,8 @@ const (
 	BatchTypeDelete = "delete"
 	// BatchTypeReset is batch type for resetting workflows
 	BatchTypeReset = "reset"
+	// BatchTypeUpdateOptions is batch type for updating the options of workflows
```

Comment:
Nit

```suggestion
	// BatchTypeUpdateOptions is batch type for updating the options of workflow executions
```

Reply:
added this but not with the commit suggestion